psd-web/app/models/user.rb

```diff
@@ -37,8 +37,12 @@ class User < ApplicationRecord
   attribute :invitation_token, :string, default: -> { SecureRandom.hex(15) }
   attribute :invited_at, :datetime, default: -> { Time.current }
 
-  def self.activated
-    where(account_activated: true)
+  def self.active
+    where(account_activated: true, deleted: false)
```

Comment:
Could chain this with the other scope, ie:

```suggestion
    where(account_activated: true).not_deleted
```

Possibly slightly easier to read?